src/pull.js

```diff
@@ -30,6 +30,35 @@ export default class Pull {
     this.side = 'remote'
   }
 
+  // Pull multiple files/dirs metadata at once, given their ids
+  async pullMany (ids: string[]) {
+    let failedIds = []
+
+    for (let id of ids) {
+      try {
+        await this.pullOne(id)
```

Comment:
I believe I also got weird issues while switching between async/await and Promise API...
Not sure why.